### PR TITLE
Doc update to deploy doryd in multi master cluster manually

### PR DESCRIPTION
**Pull request by sneharai4**
Updated manual_install_guide_hpe_3par_plugin_with_openshift_kubernetes.md to support multi-master cluster(OpenShift 3.11 and Kubernetes 1.13) with manual steps for doryd deployment.

### CODE REVIEW

**Comment by wdurairaj** [comment]
For multi-host etcd cluster, you can add this doc -- https://github.com/hpe-storage/python-hpedockerplugin/blob/master/docs/advanced/etcd_cluster_setup.md#steps-to-setup-a-3-node-secure-etcd-cluster

File: docs/manual_install_guide_hpe_3par_plugin_with_openshift_kubernetes.md [hunk 1]
```diff
@@ -263,6 +263,41 @@ $ cp /root/.kube/config /etc/kubernetes/admin.conf
...
+Run the below command to grant service account access to the hostPath plugin and the ability to run the container as root
+$ oc adm policy add-scc-to-user privileged system:serviceaccount:kube-system:doryd
+
+Run the below command to deploy doryd
+$ wget https://raw.githubusercontent.com/hpe-storage/python-hpedockerplugin/master/provisioner/OpenShift/dep-kube-storage-controller-ocp311.yaml
+$ oc create -f dep-kube-storage-controller-ocp311.yaml
+
+Run the below command to verify whether doryd is deployed successfully
+$ oc get deploy --namespace kube-system –o wide
+
```
Comment (wdurairaj) [inline docs/manual_install_guide_hpe_3par_plugin_with_openshift_kubernetes.md]: -  You can add a `kubectl get pods -n kube-system` to show the successful deployment of the doryd
-  plus add a `kubectl logs ` statement for the how to retrieve the provisioner logs.

**Comment by sneharai4** [comment]
@wdurairaj https://github.com/hpe-storage/python-hpedockerplugin/blob/master/docs/manual_install_guide_hpe_3par_plugin_with_openshift_kubernetes.md#configuring-etcd- Is already present in this doc.

File: docs/manual_install_guide_hpe_3par_plugin_with_openshift_kubernetes.md (hunk 1 above)
Comment (sneharai4) [reply docs/manual_install_guide_hpe_3par_plugin_with_openshift_kubernetes.md]: @wdurairaj Done.

**Pull request merged**